etc/common.sh

```diff
@@ -247,8 +268,49 @@ EOF
   fi
 }
 
+# ----- Install the required gpu software on the host ----- #
+install_gpu_software_base()
+{
+  echo "Installing nvidia-docker2..."
+  yum install -y epel-release
+  yum install -y moreutils runc jq  
+  yum install -y https://nvidia.github.io/nvidia-container-runtime/centos7/x86_64/nvidia-container-runtime-"$NVIDIA_CONTAINER_RUNTIME_VERSION"-1.x86_64.rpm \
+                 https://nvidia.github.io/libnvidia-container/centos7/x86_64/libnvidia-container1-"$LIBNVIDIA_CONTAINER_VERSION"-1.x86_64.rpm \
+                 https://nvidia.github.io/libnvidia-container/centos7/x86_64/libnvidia-container-tools-"$LIBNVIDIA_CONTAINER_VERSION"-1.x86_64.rpm \
+                 https://nvidia.github.io/nvidia-container-runtime/centos7/x86_64/nvidia-container-runtime-hook-"$NVIDIA_CONTAINER_RUNTIME_HOOK_VERSION"-2.x86_64.rpm \
+                 https://nvidia.github.io/nvidia-docker/centos7/x86_64/nvidia-docker2-"$NVIDIA_DOCKER_VERSION"-1.noarch.rpm 
+ 
+  #Setting up default runtime nvidia (required for kubernetes) https://github.com/NVIDIA/k8s-device-plugin#prerequisites 
+
+  jq   '{"default-runtime": "nvidia"} + .' /etc/docker/daemon.json  | sponge /etc/docker/daemon.json
```

Comment:
Is there a way to do this that does not require the installation of other packages?

Reply:
I found it like a recommendation in kubernetes documentation,
https://kubernetes.io/docs/reference/kubectl/cheatsheet/  
""jq" command useful for transformations that are too complex for jsonpath"
We can uninstall it after we run it.

Reply:
Ok. I would not remove it after, maybe it is already available on the host and the user needs it for something else.
jq is installed on its own, sponge comes from moreutils. It looks good.